pages/playback/controls/player/popup-input/input-disallow.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div>
+    <PopInput
+      :title_label="label"
+      :key_placeholder="keyplaceholder"
+      :input_check="spellchecking"
+      :invalid_text="error_text"
+      :user_key.sync="entrykey"
+      :input_width="entry_width"
+      :block="disallow_entry"
+    ></PopInput>
+  </div>
+</template>
+
+<script>
+import { PopInput } from "../../../../../dist/mantarray.common";
+// import PopInput from "@/components/playback/controls/player/PopInput.vue";
+import Vue from "vue";
+
+import uuid from "@tofandel/uuid-base62";
+Vue.use(uuid);
+
+export default {
+  components: {
+    PopInput,
+  },
+  data: function () {
+    return {
+      entrykey: "",
+    };
+  },
+  watch: {
+    entrykey() {
+      // console.log(this.entrykey);
```

Comment:
is all this code needed here?  if input is disabled is there any need for all this validation logic?  test cases (and test pages) should include just the minimum amount of code needed to run the test, that way it is easy for someone to read the test and understand what is happening